modules/op-security-with-custom-scc.adoc

```diff
@@ -0,0 +1,17 @@
+[id='op-security-with-custom-scc_{context}']
+= Security with a custom SCC
+
+You can now leverage more security with custom SCC as it's easy to migrate pipelines that need to be run or built as root. However, if you would like to lock your installation further, you can only allow running the container as the user 1000 if the pod is asking for it and the default root is not allowed.
+
+To do so you need to edit the `pipelines-scc` and modify the `runAsUser` and `seLinuxContext` section to `MustRunAs` to uid 1000.
+
+[source,yaml,subs="attributes+"]
+----
+runAsUser:
+  type: MustRunAs
+  uid: 1000
+seLinuxContext:
+  type: MustRunAs
+----
+
+The pipeline `serviceaccount` will only allow running images as user 1000 and not as root anymore.
```

Comment:
The word "will" is a red flag. Whenever possible, avoid using the future tense in technical writing. Rather, use the present tense.
```suggestion
The pipeline `serviceaccount` allows images to run only as user 1000, not as root.
```